server/services/prisonerService.js

```diff
@@ -59,15 +59,95 @@ function createPrisonerService(nomisClientBuilder) {
     }
   }
 
-  async function getComDetails(nomisClient, coms) {
-    if (!coms[0]) {
-      return null
+  async function getResponsibleOfficer(bookingId, token) {
+    logger.info(`getResponsibleOfficer: ${bookingId}`)
+    const com = await findResponsibleOfficer(bookingId, token)
```

Comment:
You throw an error for unsuspected errors in the `findResponsibleOfficer` and you are not dealing with that error here.. is the goal to let the error bubble up to the next caller?

Reply:
Yeah, all the ones we can handle (for data issues we know about) have been turned into a message, but anything unexpected like network failures will bubble up to the global handler and the standard error page.